services/minespace-web/src/components/dashboard/mine/incidents/IncidentsTable.js

```diff
@@ -156,16 +152,7 @@ export const IncidentsTable = (props) => {
                   record.mine_incident_guid
                 ),
                 state: { current: 1 },
-              });
-            }
-            return props.openModal({
```

Comment:
This is where the dead code is rooted. The above (that was previously guarded by if !IN_PROD) will always have a return, so my IDE detected the return props.openModal block as dead code, and after that, more and more started showing as dead. I have _not_ confirmed that this all still works